prac_07/myguitars.py

```diff
@@ -0,0 +1,20 @@
+from prac_06.guitar import Guitar
+import csv
+
+
+def main():
+    guitars = []
+
+    in_file = open('guitars.csv', 'r')
+    for line in in_file:
+        parts = line.strip().split(',')
+        guitar = Guitar(parts[0], int(parts[1]), float(parts[2]))
+        guitars.append(guitar)
+    in_file.close()
+
+    guitars.sort()
```

Comment:
It makes more sense to sort within the loop, this may still work for you im not sure but if it isn't try placing it like the code above

Reply:
You don't want to sort inside the loop. Just wait until the loop is finished and sort once only.